mobile/App.tsx

```diff
@@ -2,16 +2,18 @@ import { NavigationContainer } from "@react-navigation/native";
 import { createNativeStackNavigator } from "@react-navigation/native-stack";
 import React from "react";
 
+import Chess from "./src/screens/Chess";
 import ForgotPassword from "./src/screens/ForgotPassword";
 import LoginScreen from "./src/screens/LoginScreen";
 
 const Stack = createNativeStackNavigator();
 
 const App = () => (
   <NavigationContainer>
-    <Stack.Navigator initialRouteName="Login" screenOptions={{ headerShown: false }}>
+    <Stack.Navigator initialRouteName="Chess" screenOptions={{ headerShown: false }}>
```

Comment:
this file was updated in PR https://github.com/TritonSE/YSC-Mobile-Application/pull/15 so could you merge main and adjust as needed? we now have Navigator component so it would be like
```
const Navigator = () => {
  const { isLoggedIn } = useContext(AuthContext);

  return (
    <Stack.Navigator initialRouteName="Login" screenOptions={{ headerShown: false }}>
      {isLoggedIn ? (
        <Stack.Screen name="HomeScreen" component={HomeScreen} />
        <Stack.Screen name="Chess" component={Chess} />
      ) : (
        <>
          <Stack.Screen name="Login" component={LoginScreen} />
          <Stack.Screen name="ForgotPassword" component={ForgotPassword} />
        </>
      )}
    </Stack.Navigator>
  );
};
```
and then we could navigate to the Chess screen from HomeScreen when user clicks "play game" button